api/base/settings/__init__.py

```diff
@@ -22,3 +23,11 @@
     from . import defaults
     for setting in ('JWE_SECRET', 'JWT_SECRET'):
         assert getattr(local, setting, None) and getattr(local, setting, None) != getattr(defaults, setting, None), '{} must be specified in local.py when DEV_MODE is False'.format(setting)
+
+def load_institutions():
+    global INSTITUTION_ORIGINS_WHITELIST
+    from website import models
+    INSTITUTION_ORIGINS_WHITELIST = tuple(itertools.chain(*[
+        institution.domains
```

Comment:
Would you want to .lower() here too just to be safe? User input is unfortunately done most of the time by users.